README.md

```diff
@@ -397,6 +397,20 @@ _You can enable the following settings in Xcode by running [this script](resourc
   ]
   ```
 
+* <a id='no-space-inside-brackets'></a>(<a href='#no-space-inside-brackets'>link</a>) **There should be no spaces inside the brackets of collection literals.** [![SwiftFormat: spaceInsideBrackets](https://img.shields.io/badge/SwiftFormat-spaceInsideBrackets-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#spaceInsideBrackets)
```

Comment:
```suggestion
* <a id='no-space-inside-collection-brackets'></a>(<a href='#no-space-inside-brackets'>link</a>) **There should be no spaces inside the brackets of collection literals.** [![SwiftFormat: spaceInsideBrackets](https://img.shields.io/badge/SwiftFormat-spaceInsideBrackets-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#spaceInsideBrackets)
```
I wonder if we should be more specific since this is a permalink.

Reply:
I do notice that the SwiftFormat rule (`spaceInsideBrackets`) is fairly generic so that could be an argument for keeping this link more general.

Reply:
I think it makes sense for the permalink to the prose rule to be specific, even if the SwiftFormat rule implementation is a bit broader